mllib/src/main/scala/org/apache/spark/mllib/linalg/Matrices.scala

```diff
@@ -87,6 +87,9 @@ sealed trait Matrix extends Serializable {
   /** A human readable representation of the matrix */
   override def toString: String = toBreeze.toString()
 
+  /** A human readable representation of the matrix with specific lines and width */
```

Comment:
Just checked breeze's implementation. It uses max number of chars. Then we should rename `maxWidth` to `maxLineWidth`.